pkg/gds/interceptor.go

```diff
@@ -42,7 +59,7 @@ func (s *Service) serverInterceptor(ctx context.Context, in interface{}, info *g
 	// Call the handler to finalize the request and get the response.
 	var span *sentry.Span
 	if s.conf.Sentry.UsePerformanceTracking() {
-		span = sentry.StartSpan(ctx, info.FullMethod)
+		span = sentry.StartSpan(ctx, "grpc handler", sentry.TransactionName(info.FullMethod))
```

Comment:
The way I'm interpreting this from the documentation is that a `transaction` represents the full request in our case, and the `operation` is a specific part of the request handler. So if we had the need to more finely track request performance we would add more spans with the same transaction name, but for now the span is just the entire handler.

Reply:
That makes sense to me - for something like the BFF I could see breaking it down into the overall request and the request to mainnet vs testnet; but for most RPCs it's just the handler. 

One minor thing; would we just name the operation `grpc` just so it's a bit more digestible when we see it many times in Sentry? 

```suggestion
		span = sentry.StartSpan(ctx, "grpc", sentry.TransactionName(info.FullMethod))
```